src/scripts/core/models/timer.ts

```diff
@@ -0,0 +1,51 @@
+import { Container, Text as PixiText } from "pixi.js";
+import { gsap } from "gsap";
+import { eventEmitter } from "../../../event-emitter";
+
+export class Timer extends Container{
+
+    private _totalTime: number;
+    private _currentTime: number;
+    private _timeText: PixiText = new PixiText;
+
+    public get totalTime() {
+        return this._totalTime;
+    }
+
+    constructor(totalTime: number) {
+        super()
+        this._totalTime = totalTime;
+        this._currentTime = totalTime;
+        this.formatTime();
+        this._timeText.style = {
+            fill: "white"
+        };
+        this.addChild(this._timeText);
+        this.countdown();
+    }
+
+    private formatTime(): void {
+        let minutes = "0";
+        let seconds = "0";
+        this._currentTime >= 60 ? minutes = Math.floor(this._currentTime/60).toString() : minutes = '0';
```

Comment:
This seems really weird to me.
I have always seen the conditional operator used after trying to assign the variable and the value to assign after the `?`, not the variable assignment itself.
Example in this case:
`minutes = this._currentTime >= 60 ? Math.floor(this._currentTime / 60).toString() : '0';` is how it would usually be written.

It looks like it may come down to personal preference but just thought i'd bring it up

Reply:
That's a god point I had missed.
It's usually done the way Curtis points out allowing you to define and assign the variable on the same line and allowing you to make it a const instead of a let if desired.

Reply:
I didn't think it was weird but now that you've pointed it out I see that I'm doing it in a weird way. I'll make the change.